zsl/cache/redis_cache_module.py

```diff
@@ -20,12 +20,13 @@ def __init__(self, app, config):
         self._app = app
         self._config = config
 
-        redis_conf = self._config['REDIS']
+        redis_conf = self._config.get('REDIS', {})
+
         self._client = redis.StrictRedis(
-            host=redis_conf['host'],
-            port=redis_conf['port'],
-            db=redis_conf['db'] if 'db' in redis_conf else 0,
-            password=redis_conf['password'] if 'password' in redis_conf else None
+            host=redis_conf.get('host'),
+            port=redis_conf.get('port'),
+            db=redis_conf.get('db', 0),
+            password=redis_conf.get('password')
```

Comment:
I think this code makes it optional. `get` returns `None` if the key is not present.